docs/assemblies/assembly-configuring-record-encryption-filter.adoc

```diff
@@ -0,0 +1,35 @@
+// file included in the following:
+//
+// record-encryption-guide/index.adoc
+
+[id='proc-configuring-record-encryption-filter-{context}']
+= Configuring the Record Encryption filter
+
+[role="_abstract"]
+This section describes how to configure the Record Encryption filter.
+This includes configuring the filter to use the KMS that's has already been prepared.
+We will first cover how to configure each of the KMS plugins.
+We will then look at a filter configuration that uses one of those plugins.
+Finally, we will look at how to use the filter configuration in the proxy, or on Kubernetes.
```

Comment:
We should have a variable for the word Kubernetes and declare it in attributes.adoc.